src/common/lockdep.cc

```diff
@@ -261,6 +261,10 @@ int lockdep_will_lock(const char *name, int id)
 
 	assert(0);  // actually, we should just die here.
       } else {
+        BackTrace *bt = NULL;
+        if (g_lockdep >= 2 || force_backtrace) {
```

Comment:
lockdep is a bool in config_opts.h so this can't be set globally. I don't see a reason not to make it an int